<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmMain</class>
 <widget class="QMainWindow" name="frmMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Product Management Tool</string>
  </property>
  <property name="windowOpacity">
   <double>0.000000000000000</double>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="twMain">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>681</width>
      <height>381</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <property name="usesScrollButtons">
     <bool>true</bool>
    </property>
    <property name="documentMode">
     <bool>true</bool>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="movable">
     <bool>true</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Product List</string>
     </attribute>
     <widget class="QTableView" name="tblProducts">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>661</width>
        <height>311</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="QPushButton" name="btnRefresh">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>330</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Refresh</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnProductOperations">
      <property name="geometry">
       <rect>
        <x>404</x>
        <y>330</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Product Operations</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblProductInfo">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>330</y>
        <width>471</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;You can single or double click on any product that listed above.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnClear">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>330</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Search</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Search By:</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>651</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string>Name</string>
      </property>
      <widget class="QCheckBox" name="cbSearchByName">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>101</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>By Name</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txtSearchByName">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>20</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>20</y>
         <width>51</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="cbSearchByNamePattern">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>20</y>
         <width>121</width>
         <height>22</height>
        </rect>
       </property>
       <property name="currentText">
        <string>EXACTLY</string>
       </property>
       <item>
        <property name="text">
         <string>EXACTLY</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>STARTS WITH</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ENDS WITH</string>
        </property>
       </item>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>651</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string>Date</string>
      </property>
      <widget class="QCheckBox" name="cbSearchByDate">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>91</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>By Date</string>
       </property>
      </widget>
      <widget class="QDateTimeEdit" name="dteSearchByDateAddMin">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>20</y>
         <width>194</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QCheckBox" name="cbSearchByDateAdd">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>20</y>
         <width>81</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Date Add</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="cbSearchByDateUpdate">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>50</y>
         <width>81</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Date Update</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>20</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>between</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>400</x>
         <y>50</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>between</string>
       </property>
      </widget>
      <widget class="QDateTimeEdit" name="dteSearchByDateAddMax">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>20</y>
         <width>194</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QDateTimeEdit" name="dteSearchByDateUpdateMin">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>50</y>
         <width>194</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QDateTimeEdit" name="dteSearchByDateUpdateMax">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>50</y>
         <width>194</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>651</width>
        <height>51</height>
       </rect>
      </property>
      <property name="title">
       <string>Quantity</string>
      </property>
      <widget class="QCheckBox" name="cbSearchByQuantity">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>101</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>By Quantity</string>
       </property>
      </widget>
      <widget class="QComboBox" name="cbSearchByQuantityPattern">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>20</y>
         <width>121</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>LOWER</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>HIGHER</string>
        </property>
       </item>
      </widget>
      <widget class="QSpinBox" name="sbSearchByQuantity">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>20</y>
         <width>81</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="btnSearch">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>290</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSearchReset">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>200</y>
        <width>651</width>
        <height>81</height>
       </rect>
      </property>
      <property name="title">
       <string>Price</string>
      </property>
      <widget class="QCheckBox" name="cbSearchByPrice">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>91</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>By Price</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="cbSearchByPricePurchase">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>20</y>
         <width>101</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Purchase Price:</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="dsbSearchByPricePurchase">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>20</y>
         <width>81</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="cbSearchByPricePurchasePattern">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>20</y>
         <width>121</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>LOWER</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>HIGHER</string>
        </property>
       </item>
      </widget>
      <widget class="QDoubleSpinBox" name="dsbSearchByPriceSale">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>50</y>
         <width>81</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QCheckBox" name="cbSearchByPriceSale">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>50</y>
         <width>101</width>
         <height>17</height>
        </rect>
       </property>
       <property name="text">
        <string>Sale Price:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="cbSearchByPriceSalePattern">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>50</y>
         <width>121</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>LOWER</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>HIGHER</string>
        </property>
       </item>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>531</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Cambria Math</family>
      <pointsize>25</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>PRODUCT MANAGEMENT TOOL</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="lblAccountInfo">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>261</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Welcome, NULL</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>712</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actQuit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
